packages/block-library/src/search/style.scss

```diff
@@ -73,6 +73,7 @@ $button-spacing-y: math.div($grid-unit-15, 2); // 6px
 	.wp-block-search__input {
 		transition-duration: 300ms;
 		flex-basis: 100%;
+		margin-right: 10px;
```

Comment:
I think we should use a value based on our grid size. Can we use $grid-unit-10 instead of 10px?